src/pages/layer-2.js

```diff
@@ -0,0 +1,629 @@
+          and <Link to="/developers/docs/scaling/validium/">validiums</Link>.
+        </p>
+      </Content>
+
+      <Content>
+        <h2>Generalized layer 2s</h2>
+        <p>
+          Generalized layer 2s behave just like Ethereum—but cheaper. Anything
+          that you can do on Ethereum layer 1, you can also do on layer 2. Many
+          dapps have already begun to migrate to these networks, or are skipping
+          deploying to mainnet altogether.
+        </p>
+        <StyledCardGrid>
+          {layer2DataCombined
+            .filter((l2) => !l2.purpose.indexOf("universal"))
+            .map((l2, idx) => {
+              return (
+                <ProductCard
+                  key={idx}
+                  background="black"
+                  description={l2.description}
+                  url={l2.website}
+                  note={translateMessageId(l2.noteKey, intl)}
+                  name={l2.name}
+                />
+              )
+            })}
+        </StyledCardGrid>
+
+        <h2>Application specific layer 2s</h2>
+        <p>
+          Application specific layer 2s are projects that specialize in
+          optimizing for a specific application space, bringing improved
+          performance.
+        </p>
+        <StyledCardGrid>
+          {layer2DataCombined
+            .filter((l2) => l2.purpose.indexOf("universal"))
+            .map((l2, idx) => {
+              return (
+                <ProductCard
+                  key={idx}
+                  background="black"
+                  description={l2.description}
+                  url={l2.website}
+                  note={translateMessageId(l2.noteKey, intl)}
+                  name={l2.name}
+                >
+                  {l2.purpose.map((purpose) => (
+                    <Pill>{purpose}</Pill>
+                  ))}
+                </ProductCard>
+              )
+            })}
+        </StyledCardGrid>
+      </Content>
+
+      <Content>
+        <InfoBanner>
+          <h2>How to get onto a layer 2</h2>
+          <TwoColumnContent>
+            <Flex50>
+              <h4>Funds in your wallet?</h4>
+              <p>
+                If you've already got your ETH in your wallet, you'll need to
+                use a bridge to move it from Ethereum mainnet to a layer 2.{" "}
+                <Link to="/bridges/">More on bridges</Link>.
+              </p>
+            </Flex50>
+            <Flex50>
+              <h4>Funds on an exchange?</h4>
+              <p>
+                Some centralized exchanges now to offer direct withdrawals to
+                layer 2s.
+              </p>
+              {/* TODO: create cex/bridging component */}
+            </Flex50>
+          </TwoColumnContent>
+        </InfoBanner>
+      </Content>
+
+      <Content>
+        <h2>Tools and tips to be effective on layer 2</h2>
+        <TwoColumnContent>
+          <Flex50>
+            <ExpandableCard title="L2BEAT">
```

Comment:
Not a fan of the expandable cards for this type of information.

I think we could do something similar to /dapps/ here?

<img width="1544" alt="Screenshot 2022-03-14 at 12 28 40" src="https://user-images.githubusercontent.com/62268199/158172242-86515b93-2bcf-4261-bc99-b0925b756bc2.png">

Reply:
Agreed 💯! Seems silly requiring users to click to see any description of the tool.

Reply:
For sure! Will work with Jakub and Nuno on this.